config/locales/en.yml

```diff
@@ -332,6 +332,11 @@ en:
   settings:
     edit:
       title: Edit settings
+      webauthn_credentials: Credentials
```

Comment:
I call it simply "Credentials" with a note about what it is. I guess we will need docs too but trying to make it simple for the users